src/prefect/testing/standard_test_suites/blocks.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABC, abstractmethod
+from typing import Type
+
+import pytest
+
+from prefect.blocks.core import Block
+
+
+class BlockStandardTestSuite(ABC):
+    @pytest.fixture
+    @abstractmethod
+    def block(self) -> Type[Block]:
+        pass
+
+    def test_has_a_description(self, block: Type[Block]):
+        assert block.get_description()
+
+    def test_all_fields_have_a_description(self, block: Type[Block]):
+        for name, field in block.__fields__.items():
+            if Block.is_block_class(field.type_):
+                # TODO: Block field descriptions aren't currently handled by the UI, so block
+                # fields are currently excluded from this test. Once block field descriptions are
+                # supported by the UI, remove this clause.
+                continue
+            assert (
+                field.field_info.description
+            ), f"{block.__name__} is missing a description on {name}"
```

Comment:
Should it test that the description doesn't end in a period `.`

Reply:
Good question. I'll need to find out what our standard should be.

Reply:
Whether or not a description ends a in period seems to be dependent on if the description is a complete sentence/multiple sentences or not. Since it's somewhat subjective, I don't think we should write a test for it.